guides/contribution.md

```diff
@@ -0,0 +1,119 @@
+
+We encourage you to read this guide carefully and get in touch with us if you have any questions or feedback.
+
+## Index
+
+- [Proposal](#proposals)
+- [Creation of an issue based on the proposal](#issues)
+- [Updating or creating the specs needed](#specifications)
+- [Development and final review](#from-design-to-development)
+
+## Contribution journey
+
+![Contribution journey](../.github/resources/guides/contribution/contribution_journey.svg)
+
+## Proposals
+
+Before submitting a proposal, we encourage you to review:
+
+* Our existing components and guidelines to ensure that your proposal aligns with our design principles and 
+* There's no other discussion already covering the same topic. You can find all of our design guidelines and documentation on [Telefónica's brand factory](https://brandfactory.telefonica.com/d/iSp7b1DkYygv/n-a#/get-started/what-is-mistica).
+* It's not covered by the existing [Component specs](https://www.figma.com/files/1125734703130062955/project/27955986/Component-Specs?fuid=1111936175780412673) 
+* The technical viability to include the proposal in your product (A validated PRD, or confirmation that backend can serve this information)
+
+To submit a proposal, create a [new discussion in the Mística Design GitHub repository](https://github.com/Telefonica/mistica-design/discussions).
+
+Your proposal should meet the following criteria:
+
+| Approval Criteria | Description                                                                                                                                                                                                                  |
+| ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Global            | The proposal should be applicable to a broad range of use cases and scenarios, rather than being specific to a particular product or project. This ensures that the component is useful to a wider range of users and teams. |
+| Purpose           | The purpose of the proposed change needs to be well-explained. What the component is designed to solve, the benefits it provides, and any potential drawbacks or limitations.                                                |
+| Figma link        | You should provide a Figma link with examples of how the component could be used in different contexts. This helps us visualize the proposed change and understand its potential impact on the overall design system.        |
+
+Other information could help to validate the proposal earlier:
+
+- Research data.
+- Examples of current usage.
+- Examples of where the proposal could solve a problem more efficiently.
+
```

Comment:
:robot: OpenAI

The paragraph provides helpful information for validating a proposal. No changes needed.
```
Other information could help to validate the proposal earlier:

- Research data.
- Examples of current usage.
- Examples of where the proposal could solve a problem more efficiently.
```

<!-- This is an auto-generated comment by OpenAI -->